server/auth/local/verificationToken.model.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+import {Schema} from 'mongoose';
+import uuid from 'node-uuid';
+var mongoose = require('bluebird').promisifyAll(require('mongoose'));
+
+// Verification token model
+var verificationTokenSchema = new Schema({
+    userId: {
+      type: Schema.Types.ObjectId,
+      required: true,
+      ref: 'User'
+    },
+    token: {
+      type: String,
+      required: true
+    },
+    createdAt: {
+      type: Date,
+      required: true,
+      default: Date.now,
+      expires: '4h'
+    }
+});
+
+verificationTokenSchema.methods = {
+  createVerificationToken(callback) {
```

Comment:
No need to be so verbose. It is the verification token schema, so just ".create()" should imply that you are create a new entry / token